src/components/Contacts.js

```diff
@@ -0,0 +1,200 @@
+import React, { Fragment } from 'react';
+import {
+  compose,
+  withState,
+  withHandlers,
+} from 'recompose';
```

Comment:
move bracket to the same line as withHandlers. Check all import cases

Reply:
I didn't get it. Please show an example.

Reply:
TODO...

Reply:
import {
      compose,
      withState,
      withHandlers } from 'recompose';